crates/iceberg/src/catalog/mod.rs

```diff
@@ -446,6 +446,12 @@ pub enum TableUpdate {
         /// Properties to remove
         removals: Vec<String>,
     },
+    /// Remove partition specs
+    #[serde(rename_all = "kebab-case")]
```

Comment:
We don't need `rename_all` here if there are `#[serde(tag = "action", rename_all = "kebab-case")]` for the entire enum.

Reply:
This line is required as it renames the inner fields of the enum variant. When its not there, the `test_remove_partition_specs_update` test fails as `spec_ids` is wrongly serialized.

Reply:
Oh, makes sense. Thank you!